Failure generating documentation — Could not parse version component

Running scripts/document.sh with Xcode 10 fails with the following build error. This issue also affects the document.sh in the navigation SDK repository.

```
Build settings from command line:
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_REQUIRED = NO
    SWIFT_VERSION = "4.0"

note: Using new build system

** CLEAN SUCCEEDED **

note: Using new build system
note: Planning build
note: Constructing build description
Build system information
error: Invalid SWIFT_VERSION: Could not parse version component from: '"4' (in target 'Polyline-iOS')

Build system information
error: Invalid SWIFT_VERSION: Could not parse version component from: '"4' (in target 'MapboxDirections.swift-iOS')

Build system information
error: Invalid SWIFT_VERSION: Could not parse version component from: '"4' (in target 'Polyline-iOS')

Build system information
error: Invalid SWIFT_VERSION: Could not parse version component from: '"4' (in target 'MapboxDirections.swift-iOS')

** BUILD FAILED **
```

/cc @mapbox/navigation-ios

--- a/scripts/document.sh
+++ b/scripts/document.sh
@@ -29,7 +29,6 @@ DEFAULT_THEME="docs/theme"
 THEME=${JAZZY_THEME:-$DEFAULT_THEME}
 
 jazzy \
-    --podspec MapboxDirections.swift.podspec \
     --config docs/jazzy.yml \
     --sdk iphonesimulator \
     --module-version ${VERSION} \

